src/main/java/org/fogbowcloud/manager/core/models/orders/ComputeOrder.java

```diff
@@ -79,14 +92,19 @@ public UserData getUserData() {
 	public void setUserData(UserData userData) {
 		this.userData = userData;
 	}
-	
+
 	@Override
 	public OrderType getType() {
 		return OrderType.COMPUTE;
 	}
 
+	/**
+	 * These method handle and request an open order, for this, processOpenOrder
+	 * handle the Order to be ready to change your state and request the
+	 * Instance from the InstanceProvider.
+	 */
```

Comment:
I prefer to no have this method here. Instead move this logic to the Processor

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/74b2fc24f5e55b5f363881f751411bf85ddd3a2b) and [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/1b89ceb788759a1a441719c8e7f35f70949e8a82).